imports/api/questions.js

```diff
@@ -78,6 +78,9 @@ export const Questions = new Mongo.Collection('questions',
 
 // data publishing
 if (Meteor.isServer) {
+  Questions._ensureIndex({
```

Comment:
I'm not sure that we can index based on the text in a question, i.e. I don't think that would speed things up. Certainly, we should index by tags, but I don't think indexing the plain text would help to find a question that contains something faster. I assume that this would only speed things up if we  search for questions that "start with" something, rather than those that "contain" the search string.